sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md

```diff
@@ -52,7 +52,7 @@ Each extension provides three tasks you will use in your build definitions to an
 * **Run Code Analysis** task, to actually execute the analysis of the source code. 
    * This task is not required for Maven or Gradle projects, because scanner will be run as part of the Maven/Gradle build.
 * **Publish Quality Gate Result** task, to display the Quality Gate status in the build summary and give you a sense of whether the application is ready for production "quality-wise". 
-   * This tasks is optional. 
+   * This tasks is mandatory if you are using a release pipeline, with the {instance} quality gate status check pre-deployment gate, otherwise it's optional. 
```

Comment:
Is `{instance}` our custom markdown that will be resolved to SonarQube / SonarCloud when the docs are generated?
If so, is this documentation relevant for SonarQube at the moment since the new gate only seems to be for SonarCloud?

Reply:
At first sight, yes, only for SonarCloud, but since the code has been simplified, i have to test with an Azure DevOps server and a SonarQube to see if it's ok, but to my mind it should be, so the first preview might be available for SQ as well.